### PR TITLE
Added per-queue message TTL in AMQPBackend

### PR DESCRIPTION
**Pull request by shiroko**
I am targetting this branch, because this change has no influence on current configurations/features,
only add the new feature.

## Changelog

```markdown
### Added
- Added per-queue message TTL in AMQPBackend
```

## Subject

This enables to configure the "Per-Queue Message TTL" in RabbitMQ.
reference: https://www.rabbitmq.com/ttl.html

### CODE REVIEW

File: Backend/AMQPBackend.php [hunk 1]
```diff
@@ -106,6 +113,9 @@ public function initialize()
                 $args['x-dead-letter-routing-key'] = array('S', $this->deadLetterRoutingKey);
             }
         }
```
Comment (OskarStark) [inline Backend/AMQPBackend.php]: IMO we should add a blank line after this brace, what do you think @greg0ire ?

File: DependencyInjection/Configuration.php [hunk 2]
```diff
@@ -222,6 +222,12 @@ protected function getQueueNode()
 If set, the queue must be configured with this key as `routing_key`.
 EOF;
 
+        $ttl = <<<'EOF'
```
Comment (OskarStark) [inline DependencyInjection/Configuration.php]: `$ttlInfo` please in line 216 we missed that in the other PR

File: Backend/AMQPBackend.php (hunk 1 above)
Comment (greg0ire) [inline Backend/AMQPBackend.php]: I agree

File: Tests/Backend/AMQPBackendTest.php [hunk 3]
```diff
@@ -145,10 +146,52 @@ public function testInitializeWithDeadLetterExchangeAndDeadLetterRoutingKey()
...
     }
 
-    protected function getBackendAndChannelMock($recover = false, $deadLetterExchange = null, $deadLetterRoutingKey = null)
+    public function testInitializeWithTTL()
+    {
+        list($backend, $channelMock) = $this->getBackendAndChannelMock(false, null, null, self::TTL);
+
+        $channelMock->expects($this->once())
+            ->method('exchange_declare')
+            ->with($this->equalTo(self::EXCHANGE),
```
Comment (greg0ire) [inline Tests/Backend/AMQPBackendTest.php]: The first argument should be on its own line

File: Tests/Backend/AMQPBackendTest.php [hunk 4]
```diff
@@ -145,10 +146,52 @@ public function testInitializeWithDeadLetterExchangeAndDeadLetterRoutingKey()
...
+                   $this->equalTo('direct'),
+                   $this->isType('boolean'),
+                   $this->isType('boolean'),
+                   $this->isType('boolean')
+             );
+        $channelMock->expects($this->once())
+            ->method('queue_declare')
+            ->with($this->equalTo(self::QUEUE),
```
Comment (greg0ire) [inline Tests/Backend/AMQPBackendTest.php]: same here

File: Tests/Backend/AMQPBackendTest.php (hunk 3 above)
Comment (OskarStark) [inline Tests/Backend/AMQPBackendTest.php]: Good catch

**Comment by greg0ire** [comment]
どうもありがとう @shiroko !

**Comment by shiroko** [comment]
Thanks!

**Pull request merged**